### DISCUSSION TITLE
Feature request: Adding tiled diffusion upscaler for image 2 image and change location from dropdown script to always visible tab

### DISCUSSION THREAD

**Discussion by Kking1111** [AIrjen/OneButtonPrompt#116] [Ideas]
I think it's a great idea to add this multi diffusion upscaler extension as an option. I am assuming it is possible because already control net tiling and ultimate sd upscalers are available optionally.

Also, it would be great to make this extension permanently visible, rather than to select from a dropdown menu of scripts. This will also allow it to be used with other scripts.

**Comment by AIrjen** [comment]
Hi! Yes, making it a full extension instead of a script would be nice. And a lot of work. I might consider this if I ever do a 2.0 version.

For multi diffusion upscaler extension to be included in the upscaler options... I can put it on the todo list somewhere, but I am currently not inclined to pick this up.

**Comment by Kking1111** [comment]
Thank you for you quick response, I appreciate it.
